### DISCUSSION TITLE
Rounded relative parent with absolute child and overflow is not working properly

### DISCUSSION THREAD

**Discussion by teddybee** [tailwindlabs/tailwindcss#10137] [Help] [answered]
I have small react ap, I would leike to keep the page fixed height, basically a search panel and a result panel and a footer. 
I would like to achieve to have the result panel rounded and has right slide if needed.
In react, I can do that, but scrolling down a long content, some times it simply disappears and continue to show lower scrolling.
In plain html, if I put rounded and overflow-y-auto and/or overflow-hidden to the parent, shows nothing.
[https://play.tailwindcss.com/V3UWXFyreU](https://play.tailwindcss.com/V3UWXFyreU)
I can do a react version of strange behaviour as well if you need.

**Comment by wongjn** [comment, marked as answer]
I'm not quite sure what you are after, but there is a modified version of the Tailwind Play but with scrolling sections: https://play.tailwindcss.com/QzjreqkeFz

**Comment by teddybee** [comment]
Almost exactly this. Thank you very much.
